src/request/credit-card.ts

```diff
@@ -16,14 +45,46 @@ const issuerIdNumberRegex = /^[0-9]{6}$/;
 const last4Regex = /^[0-9]{4}$/;
 const tokenRegex = /^(?![0-9]{1,19}$)[\u0021-\u007E]{1,255}$/;
 
+/**
+ * The credit card information for the transaction being sent to the web service.
+ */
 export default class CreditCard implements CreditCardProps {
+  /**
+   * The issuer ID number for the credit card. This is the first 6 digits of
+   * the credit card number. It identifies the issuing bank.
+   */
```

Comment:
Is there a way to inherit this documentation so we don't need to have it twice?

Reply:
Not that I'm aware of.

Reply:
I think `/** @inheritdoc */` on the property will do it based on my testing. There may be a better way.

Reply:
Nice!  I've made the change here: https://github.com/maxmind/minfraud-api-node/pull/239/commits/0cd150ae187e3a3366599ac4b2ebc6b49c33d631